Resources/Private/Frontend/JavaScript/Features/ContentTree.ts

```diff
@@ -49,30 +50,30 @@ function createContentTreeHTML() {
   contentTreeHeader.innerHTML = '<label class="content-tree__headline">Content tree</label><i class="fa fa-solid fa-angle-down content-tree__toggle"></i>';
   contentTreeHeader.onclick = initContentTreeToggle;
 
-  const contentTreeHeadline = contentArea.querySelector('.typo3-docheader-pagePath + strong') ? contentArea.querySelector('.typo3-docheader-pagePath + strong').innerHTML : (contentArea.querySelector('.t3js-title-inlineedit') ? contentArea.querySelector('.t3js-title-inlineedit').textContent : 'Content Tree');
+  const contentTreeHeadline = contentArea?.querySelector<HTMLElement>('.typo3-docheader-pagePath + strong') ? contentArea?.querySelector<HTMLElement>('.typo3-docheader-pagePath + strong')?.innerHTML : (contentArea?.querySelector<HTMLElement>('.t3js-title-inlineedit') ? contentArea?.querySelector<HTMLElement>('.t3js-title-inlineedit')?.textContent : 'Content Tree');
```

Comment:
This line is pretty confusing. It includes nested checks. Would it be possible to use `if`-clauses for the presence checks?
Or even move it into a separate generator function?

The other option would be to move it to the next rows - just like you did in the line 114.

Reply:
done :)